next/app/components/common/GenerateButton/GenerateButton.module.css

```
@@ -0,0 +1,42 @@
+.generateButton {
+  background-color: #edc3e7;
+  border: 1px solid black;
+  color: black;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 20px;
+  font-weight: bold;
+  cursor: pointer;
+  border-radius: 12px;
+  transition:
+    background-color 0.3s,
+    color 0.3s,
+    border-color 0.3s;
+  height: 32px; /* 固定の高さを設定 */
+  width: 96px; /* 固定の幅を設定 */
+}
+.loader {
+  border: 4px solid #ffffff; 
+  border-top: 4px solid #cccccc; 
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  animation: spin 3s linear infinite;
+  margin: auto;
+}
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(720deg);
+  }
+}
+
```

「ホバーするときは、色の反転ではなく色が薄くなるようにしてほしい。」とのことだったのでhover時の挙動も追加しておきましょう。（色の変更はあるかもですが）
```suggestion
.generateButton:hover {
  background-color: #edc3e77f;
  color: #2C3844;
  border-color: #2C3844;
}
```